ethportal-api/src/types/consensus/beacon_state.rs

```diff
@@ -0,0 +1,312 @@
+#[cfg(test)]
+use crate::consensus::fork::ForkName;
+use crate::consensus::{
+    body::{Checkpoint, Eth1Data},
+    execution_payload::{ExecutionPayloadHeaderBellatrix, ExecutionPayloadHeaderCapella},
+    header::BeaconBlockHeader,
+    header_proof::HistoricalSummary,
+    participation_flags::ParticipationFlags,
+    pubkey::PubKey,
+    sync_committee::SyncCommittee,
+};
+use discv5::enr::k256::elliptic_curve::consts::{U1099511627776, U2048, U4, U65536, U8192};
+use ethereum_types::H256;
+use jsonrpsee::core::Serialize;
+use serde::Deserialize;
+use serde_this_or_that::as_u64;
+use serde_utils;
+use ssz::{Decode, DecodeError, Encode};
+use ssz_derive::{Decode, Encode};
+use ssz_types::{typenum::U16777216, BitVector, FixedVector, VariableList};
+use std::sync::Arc;
+use superstruct::superstruct;
+use tree_hash::{Hash256, TreeHash};
+use tree_hash_derive::TreeHash;
```

Comment:
Don't we have a clippy lint for ordering imports into groups? eg. `std` first, exterior second, internal third

Reply:
Not sure what ordering we use but this can be changed in the config. I think now it is internal first, std next and external latest.

Reply:
Based on the `rustfmt` [documentation](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports), the default is (what we use) to preserve the source file's import groups.

Alternative, which we could enable, is closer to what @njgheorghita said:
1. `std`, `core` and `alloc`,
2. external crates,
3. `self`, `super` and `crate` imports.